packages/cadl-rlc-test/test/anomalyDetector/cadl-output/src/anomalyDetector.ts

```diff
@@ -0,0 +1,46 @@
+import { getClient, ClientOptions } from "@azure-rest/core-client";
+import { KeyCredential } from "@azure/core-auth";
+import { AnomalyDetectorClient } from "./clientDefinitions";
+
+/**
+ * Initialize a new instance of the class AnomalyDetectorClient class.
+ * @param Endpoint type: string
+ * @param ApiVersion type: "v1.1"
+ * @param credentials type: KeyCredential
+ */
+export default function createClient(
+  Endpoint: string,
+  ApiVersion: "v1.1",
```

Comment:
I wonder if this is expected ?

Reply:
I guess it is because the CADL doesn't have a default value for ApiVersion. If it had in CADL ApiVersion = "v1.1" it should take this out of the required parameters.

However, heads up that we'll need to change this, as this is a Service Default, not a Client Default. Client Default is not yet supported in CADL, and it is in the works. Once it is supported we'll need to change our generation logic

There are 2 default values:

- Service Default [Supported on CADL today] -> Doesn't have any effect on the generated client, maybe just docs. This tells what the service is going to assume if the value is not provided.

- Client Default [Not supported] -> This does have an effect on the generated client, and tells what the client will send if the user didn't explicitly set the value.